header/p2p/options.go

```diff
@@ -44,6 +48,10 @@ func (p *ServerParameters) Validate() error {
 	if p.MaxRequestSize == 0 {
 		return fmt.Errorf("invalid max request size: %d", p.MaxRequestSize)
 	}
+	if p.RequestDuration == 0 {
+		return fmt.Errorf("invalid RequestDuration for session: "+
```

Comment:
```suggestion
		return fmt.Errorf("invalid request duration for session: "+
```

just more consistent with the other err messages